ldcpy/__init__.py

```diff
@@ -1,3 +1,13 @@
+#!/usr/bin/env python3
+# flake8: noqa
+""" Top-level module for ldcpy. """
+from pkg_resources import DistributionNotFound, get_distribution
+
 from .metrics import DatasetMetrics, DiffMetrics
 from .plot import plot
 from .util import check_metrics, collect_datasets, compare_stats, open_datasets
+
+try:
+    __version__ = get_distribution(__name__).version
+except DistributionNotFound:  # pragma: no cover
+    __version__ = 'unknown'  # pragma: no cover
```

Comment:
@pinarda, with these lines, we don't need to hard-code the version in `setup.py`. The version will be retrieved at runtime from the latest git tag + git commit information. 

When it comes to releasing newer version of the package, all we need is to create and push a new tag to GitHub.